test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -746,29 +724,4 @@ public void Ctor_WrapsCustomTransportWhenCachePathOnOptions()
         var cachingTransport = Assert.IsType<CachingTransport>(_fixture.SentryOptions.Transport);
         _ = Assert.IsType<FakeTransport>(cachingTransport.InnerTransport);
     }
-
-    [Fact]
-    public async Task SentryClient_WithCachingTransport_RecordsDiscardedEvents()
```

Comment:
This test was irrelevant, as the event gets discarded when captured, not when passed to the transport.  The transport is never invoked anyway, as we substituted the background worker.

It's also redundant, as we already test that we record discarded events correctly.